heron/packing/src/java/org/apache/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -367,16 +410,16 @@ private ByteAmount getContainerRamHint(Map<Integer, List<InstanceId>> allocation
 
     return TopologyUtils.getConfigWithDefault(
         topologyConfig, org.apache.heron.api.Config.TOPOLOGY_CONTAINER_RAM_REQUESTED,
-        NOT_SPECIFIED_NUMBER_VALUE);
+        NOT_SPECIFIED_BYTE_AMOUNT);
   }
 
   /**
-   * Check whether the PackingPlan generated is valid
+   * Check whether the PackingPlan configured each instance to have at least minimum required RAM
```

Comment:
There might be more checks in the function, like CPU and disk and maybe others. I feel validatePackingPlan() makes more sense here.